Game/Source/GDKTestGyms/GDKTestGymsGameInstance.h

```diff
@@ -8,6 +8,9 @@
 
 #include "GDKTestGymsGameInstance.generated.h"
 
+#define OUTPUT_NFR_SCENARIO_LOGS !WITH_EDITOR
```

Comment:
What if we make this a small inline function? If this file isn't included it OUTPUT_NFR_SCENARIO_LOGS will default to 0

Reply:
oh man - i thought it wouldn't compile because it was #if instead of #ifdef.
yeah can make this safer